config/initializers/rails_admin.rb

```diff
@@ -125,4 +125,10 @@
   end
 
   config.compact_show_view = false
+
+  # Non-rails-admin links
+  config.navigation_static_label = 'Workflows'
+  config.navigation_static_links = {
+    'Activity Insight Open Access Workflow' => '/activity_insight_oa_workflow'
```

Comment:
I'm not 100% on this.  It works, but it's generated by RailsAdmin so the link it creates opens the page in a new tab.  Also, you can't set the host this early in the initialization and the routes haven't been loaded, so you cannot use url helpers.

Reply:
Yeah, it's a bummer that this isn't more configurable. I took a look, and I don't see any other reasonable way to add this to the navigation. If we wanted, we could probably add in a little javascript hack to at least make the link open in the same tab.